src/VSCodeExtension/src/formatter/rules/operation.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Formats operation declarations.
+ *
+ * @param code incoming code
+ *
+ * @example `operation   Foo(q:Qubit,  n   :Int)   :  Bool` ->
+ * `operation Foo (q : Qubit, n : Int) : Bool`

```

Comment:
I'd generally suggest not including a space between the operation name and its inputs.

+@swernli for style guide discussion.

Reply:
Yup! That's one of the open questions, and actually the next one I think we could tackle.

Reply:
Deferring to teams channel for discussion.